xiplot/tabs/plugins.py

```diff
@@ -0,0 +1,477 @@
+                                className="button",
+                            ),
+                        ),
+                        title="Install a Plugin",
+                        css_class="dash-dropdown",
+                    ),
+                    html.Br(),
+                    layout_wrapper(
+                        component=FlexRow(
+                            dcc.Dropdown(
+                                get_loaded_plugin_options(),
+                                id="loaded_plugins",
+                                clearable=False,
+                                searchable=False,
+                                placeholder=(
+                                    "Check the list of already loaded plugins"
+                                ),
+                            ),
+                            html.Button(
+                                "Reload χiplot",
+                                id="plugin-reload-button",
+                                n_clicks=0,
+                                className="button",
+                                disabled=True,
+                            ),
+                            dmc.Modal(
+                                title="Do you really want to reload χiplot?",
+                                id="plugin-reload-modal",
+                                zIndex=10000,
+                                children=[
+                                    dmc.Text(
+                                        "Please ensure that you have"
+                                        " downloaded your plots and data so"
+                                        " that nothing is lost. Any plugins"
+                                        " that you have installed will become"
+                                        " loaded after the reload."
+                                    ),
+                                    dmc.Space(h=20),
+                                    dmc.Group(
+                                        [
+                                            dmc.Button(
+                                                "Reload",
+                                                id="plugin-reload-accept",
+                                            ),
+                                            dmc.Button(
+                                                "Cancel",
+                                                color="red",
+                                                variant="outline",
+                                                id="plugin-reload-cancel",
+                                            ),
+                                        ],
+                                        position="right",
+                                    ),
+                                ],
+                            ),
+                        ),
+                        title="Loaded Plugins",
+                        css_class="dash-dropdown",
+                    ),
```

Comment:
After installing a plugin I immediately opened the "loaded plugins list" and it wasn't there. It only appeared after a reload. This does make sense, but I think calling the list "active plugins" instead of "loaded plugins" would help with this confusion.

```suggestion
                    layout_wrapper(
                        component=FlexRow(
                            dcc.Dropdown(
                                get_loaded_plugin_options(),
                                id="loaded_plugins",
                                clearable=False,
                                searchable=False,
                                placeholder=(
                                    "Check the list of activated plugins"
                                ),
                            ),
                            html.Button(
                                "Reload χiplot",
                                id="plugin-reload-button",
                                n_clicks=0,
                                className="button",
                                disabled=True,
                            ),
                            dmc.Modal(
                                title="Do you really want to reload χiplot?",
                                id="plugin-reload-modal",
                                zIndex=10000,
                                children=[
                                    dmc.Text(
                                        "Please ensure that you have"
                                        " downloaded your plots and data so"
                                        " that nothing is lost. Any plugins"
                                        " that you have installed will become"
                                        " activated after the reload."
                                    ),
                                    dmc.Space(h=20),
                                    dmc.Group(
                                        [
                                            dmc.Button(
                                                "Reload",
                                                id="plugin-reload-accept",
                                            ),
                                            dmc.Button(
                                                "Cancel",
                                                color="red",
                                                variant="outline",
                                                id="plugin-reload-cancel",
                                            ),
                                        ],
                                        position="right",
                                    ),
                                ],
                            ),
                        ),
                        title="Active Plugins",
                        css_class="dash-dropdown",
                    ),
```